src/components/GettingStartedModal/Preamble.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactElement } from "react";
+import { NavigationButtons, Step, StepProps } from ".";
+import { Button } from "../Button";
+
+export const Preamble: Step = {
+  /*
+   * Defines the preamble step. Welcomes users to the site. 
+   */
+  title: "Welcome to Cheroke Language Exercises!",
+  commitState: () => {},
+  Component: FirstStepComponent,
+};
+
+function FirstStepComponent({
+  /*
+   * The first component of the wizard. Advances to the next step or redirects to FAQ page.
+   */
+  wizardState,
+  goToNextStep,
+  goToPreviousStep,
+  exitWizard
+}: StepProps): ReactElement {
+  return (
+    <div>
+      This website is dedicated to helping users develop knowledge of the Cherokee language.
+      <br></br>
+      <br></br>
+      Code of Conduct:
+      <br></br>
+        - Filler text 1.
+        <br></br>
+        - Filler text 2.
+        <br></br>
+        - Filler text 3.
+      <br></br>
```

Comment:
I know this is filler, but unordered lists should be written with the `ul` (Unordered List) and `li` (List item) tags, FWIW.

Reply:
Also, this many `<br/>`s is scary -- maybe we need some components with padding/margins?

Reply:
I temporarily changed it to the unordered list, do you want me to add a component for the code of conduct?